src/cli/internal.rs

```diff
@@ -161,6 +161,12 @@ impl<'i> Internal<'i> {
         log!("process removed (id={})", self.id);
     }
 
+    pub fn flush(&mut self) {
+        self.runner.flush(self.id);
```

Comment:
Can add a temporary program stop here when trying to use a external server, as the code for flushing external logs was not added in this PR, and could confuse users.